.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 14.19
```

Comment:
I've had to use a later version of Node to run the version of Playwright that has the Clock API, I understand we most likely don't want to diverge too much, but this felt like an okay compromise. In particular since later versions of Playwright also come with UI mode which is great for TDD, so that's the version I've had personally installed for a while, but tests wouldn't pass on it before, now they do, so could be a good occasion to upgrade

Reply:
16 is atm the version on jenkins for build, and also the version on the server for nodejs runtime

Reply:
@simon-debruijn Sorry just to double check, are you saying I should use 16 here or that we should push for 18 on the actual app?